### PR TITLE
Convert `hide.modules.css` to CSS modules

### LINKED ISSUE #40571: Migrate global css in `frontend/src/metabase/css/core/hide.module.css`

### PR DESCRIPTION
**Pull request by oisincoveney**
### Description

Describe the overall approach and the problem being solved.

### How to verify

Describe the steps to verify that the changes are working as expected.

1. New question -> Sample Dataset -> ...
2. ...

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist

- [ ] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: frontend/src/metabase/css/core/hide.module.css [hunk 1]
```diff
@@ -1,87 +1,15 @@
-:global(.hide) {
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/css/core/hide.module.css:1]: Good finds. I really don't understand how my `Find in files` in webstorm didn't pick up on theese

File: frontend/src/metabase/css/core/hide.module.css [hunk 2]
```diff
@@ -1,87 +1,15 @@
...
 
-:global(.show) {
-  display: inherit;
-}
-
-:global(.hidden) {
-  visibility: hidden;
-}
-
-:global(.sm-show),
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/css/core/hide.module.css:13]: Should be sorted. check out `Button.tsx` for the solution to use the dynamic breakpoints

File: frontend/src/metabase/css/core/hide.module.css [hunk 3]
```diff
@@ -1,87 +1,15 @@
-:global(.hide) {
+.hide {
   display: none !important;
 }
 
-:global(.show) {
```
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/css/core/hide.module.css:5]: I moved the classes that use .show from `TableInteractive` to the `hide.modules.css`. Let me know if there's a better way to do this

File: frontend/src/metabase/css/core/hide.module.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (oisincoveney) [reply frontend/src/metabase/css/core/hide.module.css:1]: Sorted

**Review by deniskaber — APPROVED** [review]

**Pull request merged**